docs/source/building_an_experiment.rst

```diff
@@ -4,14 +4,227 @@ Building an Experiment
+
+**Sequence (integration)**  
+
+Made up of pulses with a specific spacing, at a specific frequency, and with a specified receive time 
+following the transmission (to gather information from the number of ranges specified).
+
+**Averaging period (integration time)**  
+
+A time where the sequences are repeated to gather enough information to average and reduce the effect of 
+spurious emissions on the data. These are defined by either number of sequences, or a length of time during 
+which as many sequences as possible are transmitted.
+
+**Scan**  
+
+A time where the averaging periods are repeated, often with the pulses mixed to look in different beam 
+directions with each averaging period. A scan is defined by the number of beams or integration times.
+
+Interfacing types
+-----------------
+
+Knowing the basic building blocks of a SuperDARN-style experiment, the following types of interfacing are possible:
+
+**1. SCAN**   
+
+The scan by scan interfacing allows for slices to run a scan of one slice, followed by a scan of the second. The scan mode of interfacing typically means that the slice will cycle through all of its beams before switching to another slice.
+
+**2. INTTIME**   
+
+This type of interfacing allows for an integration period to run for one slice, before switching to another. This type of interface effectively creates an interleaving scan where the scans for different slices are run simultaneously, however the pulse sequences are alternated integration time by 
+integration time rather than run concurrently.
+
+**3. INTEGRATION**   
+
+Integration interfacing allows for pulse sequences defined in the slices to alternate sequence by sequence each other within an integration period. Slices which are interfaced in this manner must share the same INTT and INTN values for this to work. It's important to remember that each sequence 
+only averages with sequences from the same slice. 
+
+**4. PULSE**   
+
+Pulse interfacing allows for pulse sequences to be run together concurrently. Slices will have their pulse sequences mixed and layered together so that the data transmits at the same time. Slices of different frequencies can be 
+mixed simultaneously and slices of different pulse sequences can also run together at the cost of having more blanked samples.
+
+Let's look at some examples of common experiments that can easily be separated into multiple slices. 
+
+In a CUTLASS-style experiment, the pulse in the sequence is actually two pulses of differing transmit frequency. This is a 'quasi'-simultaneous multi-frequency experiment. To build this experiment, two slices can be PULSE interfaced. The pulses from both slices are combined into a single sequence and data from those integrations are used for both slices (filtering the raw data separates the frequencies). 
```

Comment:
Is 'CUTLASS-style' widely known to mean 'pseudo-stereo'? 
That may just be what we've been calling it.